openedx/core/djangoapps/learner_pathway/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class LearnerPathwayConfig(AppConfig):
+    default_auto_field = 'django.db.models.BigAutoField'
+    name = 'learner_pathway'
```

Comment:
we could add a verbose_name here as well

Reply:
done